packages/localizations/src/be-BY.ts

```diff
@@ -547,7 +547,7 @@ export const beBY: LocalizationResource = {
     },
   },
   socialButtonsBlockButton: 'Працягнуць з дапамогай {{provider|titleize}}',
-  socialButtonsBlockButtonManyInView: 'Увядзіце дадатковыя спосабы ўваходу',
+  socialButtonsBlockButtonManyInView: undefined,
```

Comment:
@panteliselef  Why don't we do the following for all localizations?
```suggestion
  socialButtonsBlockButtonManyInView: '{{provider|titleize}}',
```

Reply:
@anagstef if left undefined it will use the one defined in `en-US` right ?

Reply:
Yes, but there is a risk that someone will try to translate this wrong in the future again.

Reply:
i'm confused, but Typescript will not let them.